### PR TITLE
Vertical-navbar-modified

### LINKED ISSUE #886: Vertical navbar links
Can I make the navbar links verticals instead of horizontal?

Navbar
 
I think it will look nice If I made it vertical. Can you assign me this issue, please? @anushbhatia

### PR DESCRIPTION
**Pull request by MennaMagdy2019**
fixed #886 
I modified the Vertical navbar links as you told 
as reference :https://girlscriptboilerplate.smaranjitghose.com/ 
and fixed the merge conflicts
 and added level3 & partially approved
Can you review it please sir? @sameersrivastava13

### CODE REVIEW

**Comment by smaranjitghose** [comment]
## Your changes:

current_nav

## Reference:

ref_nav

**Comment by MennaMagdy2019** [comment]
Can you tell me please How Can I fix the merge conflict @smaranjitghose

**Comment by MennaMagdy2019** [comment]
> ## Your changes:
> current_nav
> 
> ## Reference:
> ref_nav
sorry It happened some error

**Comment by MennaMagdy2019** [comment]
I already do this but I can't solve the fixed merge conflict @smaranjitghose 
Vertical navbar

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect, useRef } from "react";
 import { NavLink, useHistory } from "react-router-dom";
 import { Link } from "react-scroll";
-import styles from "./header.module.scss";
```
Comment (smaranjitghose) [inline src/components/Header/Header.jsx]: Please don't change the name of the file

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
> I already do this but I can't solve the fixed merge conflict @smaranjitghose
> Vertical navbar

- The texts should be centre aligned w.r.t the screen (top and bottom). There is too much space at the top
- Increase the size of the text

**Comment by smaranjitghose** [comment]
> Can you tell me please How Can I fix the merge conflict @smaranjitghose

- Pull the changes from the upstream repo(this one)
- Integrate them into your feature branch
- Add/modify your changes accordingly
- Make a Squash Commit
- Update the PR

**Comment by ashuvssut** [comment]
DO NOT MERGE this PR yet

see my review at PR #900.
same applied to this one

File: src/App.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Route, Switch, Redirect } from "react-router-dom";
 import { withRouter } from "react-router";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
-import NotFound from "./pages/404/NotFound";
+import NotFound from "./pages/404/notFound";
```
Comment (ashuvssut) [inline src/App.jsx:6]: DO NOT MERGE this PR yet

see my review at PR #900.
same applied to this one

**Pull request closed without merging**